project-addons/vt_flask_middleware/customer.py

```diff
@@ -33,6 +33,9 @@ class Customer(SyncModel):
     commercial_id = ForeignKeyField(Commercial, on_delete='CASCADE', null=True)
     odoo_id = IntegerField(unique=True)
     lang = CharField(max_length=5, null=True)
+    type = CharField(max_length=30)
```

Comment:
null=True

Reply:
Esto seria para el caso en el que fueran empresas ¿No?

Reply:
Por lo qu eveo en el código propuesto:
` if not vals['is_company']:
 +            vals.update({"type": partner.type, "parent_id": partner.parent_id.id, "email": partner.email})`

Sino es empresa no se rellena el type por lo que no puede ser requerido en la base de datos.

Reply:
Cierto, ahora subo el cambio